packages/office-ui-fabric-react/src/components/Dialog/Dialog.styles.ts

```diff
@@ -31,3 +33,30 @@ export const getStyles = (props: IDialogStyleProps): IDialogStyles => {
     ]
   };
 };
+
+export const getMergedDialogContentStyles = (
+  dialogContentProps?: IDialogContentProps,
+  dialogDraggableClassName?: string
+): IStyleFunctionOrObject<IDialogContentStyleProps, IDialogContentStyles> => {
+  const headerStyle = {
+    header: [
+      dialogDraggableClassName,
+      dialogDraggableClassName && {
+        cursor: 'move'
+      }
+    ]
+  };
+
+  if (dialogContentProps && dialogContentProps.styles) {
+    if (typeof dialogContentProps.styles === 'function') {
+      const styles = dialogContentProps.styles;
+      return (props: IDialogContentStyleProps) => {
+        return mergeStyleSets(styles(props), headerStyle);
+      };
+    } else {
+      return mergeStyleSets(dialogContentProps.styles, headerStyle);
```

Comment:
reverse these; headerStyle should be less specific than passed in ones, I believe.